src/2ls/instrument_goto.cpp

```diff
@@ -154,4 +154,6 @@ void instrument_gotot::operator()(goto_modelt &goto_model)
   }
 
   goto_model.goto_functions.update();
+  if(options.get_unsigned_int_option("verbosity") == 10)
+    goto_model.output(std::cout);
```

Comment:
I think that it would make more sense to run this either from `summary_checker_base::instrument_and_output` or from `2ls_parse_options`.

Reply:
`instrument_and_output` would definitely make more sense, good point. However, from what I can see, `summary_checker_base` doesn't have access to `ui_message_handler` which holds the verbosity. So either we can pass the handler to `summary_checker_base` or add verbosity to `optionst` which `summary_checker_base` can access.

Reply:
Just pass the verbosity level as a parameter of `instrument_and_output`.